Build/Build.cs

```diff
@@ -282,9 +278,7 @@ void ReportTestOutcome(params string[] globFilters)
 
             var testCombinations =
                 from project in projects
-                let frameworks = project.GetTargetFrameworks()
```

Comment:
special windows workaround gone 🎉

Reply:
`XUnit2.Specs` still has a net47 target, so this change breaks running the `TestFrameworks` on non-windows.
![image](https://github.com/fluentassertions/fluentassertions/assets/919634/19f89ef9-c4af-42b1-85b6-48e9020c33cb)

Our Linux+mac pipelines only run `UnitTests`, so that's why they don't fail.
From #1525 it seems that it's relevant to test xunit on both TFMs.

Reply:
Didn't noticed that :)